src/tree_map.h

```diff
@@ -215,50 +215,6 @@ static inline void SetTreeGrowth(TileIndex t, uint g)
 	SB(_m[t].m5, 0, 3, g);
 }
 
-/**
```

Comment:
If you remove it, also set it to 0 in the map structure in `AfterLoadGame` just to prevent nasty surprises in the future.

You can guard it with `IsSavegameVersionBeforeOrAt(SLV_<whatever_it_currently_is>)`, which would mean that it will set the field to 0 for every savegame until the next savegame bump is made. This way you will immediately see the result in (network) savegame size, at a tiny cost during loading.

While you're at it, in afterload.cpp around line 2352 (within a `IsSavegameVersionBefore(SLV_135)`) the counter gets read and written. That read can go and the counter can be removed from the bit-or. I wouldn't complicate the clearing logic of the counter by checking whether it is after this savegame version though.